### PR TITLE
Show all beers

### PR DESCRIPTION
**Pull request by Patrik200414**
Creating show all beers function

### CODE REVIEW

**Comment by Patrik200414** [comment]
Creating show all beers

File: showAll.js [hunk 1]
```diff
@@ -6,6 +6,11 @@
 const getBeers = require('./data')
 
 function main() {
+    const beers = getBeers();
```
Comment (ZsoltEmber) [inline showAll.js:9]: asdasd

Comment (ZsoltEmber) [reply showAll.js:9]: mégasdabb

Comment (Patrik200414) [inline showAll.js:9]: Beers

**Comment by ZsoltEmber** [comment]
"asd

**Pull request merged**